tests/min_tests.py

```diff
@@ -92,6 +92,16 @@ def run_testsuit():
 
 
 if __name__ == "__main__":
+
+    # testing import submodules
+    from monai.utils.module import load_submodules
+
+    _, err_mod = load_submodules(sys.modules["monai"], True)
+    if err_mod:
+        # expecting that only engines and handlers are not imported
+        assert sorted(err_mod) == ["monai.engines", "monai.handlers"]
```

Comment:
May I know what's the logic here? Because these 2 modules depend on ignite?
Thanks.

Reply:
yes they couldn’t be imported in the minimal env